website/docs/r/search_deployment.html.markdown

```diff
@@ -1,57 +1,99 @@
 ---
 layout: "mongodbatlas"
-page_title: "MongoDB Atlas: search deployment"
-sidebar_current: "docs-mongodbatlas-resource-search-deployment"
+page_title: "MongoDB Atlas: mongodbatlas_search_deployment"
+subcategory: "docs_mongodbatlas_search_deployment_resource"
 description: |-
-    Provides a Search Deployment resource.
+    "Provides a Search Deployment resource."
 ---
 
 # Resource: mongodbatlas_search_deployment
 
-`mongodbatlas_search_deployment` provides a Search Deployment resource. The resource lets you create, edit and delete dedicated search nodes in a cluster.
 
--> **NOTE:** For details on supported cloud providers and existing limitations you can visit the [Search Node Documentation](https://www.mongodb.com/docs/atlas/cluster-config/multi-cloud-distribution/#search-nodes-for-workload-isolation).
 
--> **NOTE:** Only a single search deployment resource can be defined for each cluster.
+## Example Usages
 
+```terraform
+resource "mongodbatlas_project" "example" {
+  name   = "project-name"
+  org_id = var.org_id
+}
 
-## Example Usage
+resource "mongodbatlas_advanced_cluster" "example" {
+  project_id   = mongodbatlas_project.example.id
+  name         = "ClusterExample"
+  cluster_type = "REPLICASET"
+
+  replication_specs {
+    region_configs {
+      electable_specs {
+        instance_size = "M10"
+        node_count    = 3
+      }
+      provider_name = "AWS"
+      priority      = 7
+      region_name   = "US_EAST_1"
+    }
+  }
+}
 
-```terraform
-resource "mongodbatlas_search_deployment" "test" {
-  project_id = "PROJECT ID"
-  cluster_name = "NAME OF CLUSTER"
+resource "mongodbatlas_search_deployment" "example" {
+  project_id   = mongodbatlas_project.example.id
+  cluster_name = mongodbatlas_advanced_cluster.example.name
   specs = [
     {
       instance_size = "S20_HIGHCPU_NVME"
-      node_count = 2
+      node_count    = 2
     }
   ]
 }
+
+data "mongodbatlas_search_deployment" "example" {
+  project_id   = mongodbatlas_search_deployment.example.project_id
+  cluster_name = mongodbatlas_search_deployment.example.cluster_name
+}
+
+output "mongodbatlas_search_deployment_id" {
+  value = data.mongodbatlas_search_deployment.example.id
+}
 ```
 
-## Argument Reference
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `cluster_name` (String) Label that identifies the cluster to return the search nodes for.
+- `project_id` (String) Unique 24-hexadecimal character string that identifies the project.
+- `specs` (Attributes List) List of settings that configure the search nodes for your cluster. (see [below for nested schema](#nestedatt--specs))
+
+### Optional
+
+- `timeouts` (Attributes) (see [below for nested schema](#nestedatt--timeouts))
```

Comment:
nit: can you just check that we are not losing some relevant information here with the change of timeouts field compared to how it was before?